_data/hosts.yml

```diff
@@ -130,6 +130,39 @@
             version: 5.6.11
             semver: 5.6.11
 -
+    name: CloudAccess
+    url: 'http://www.cloudaccess.net/'
+    type: shared
+    php70: 0
```

Comment:
This key doesn't need to be there. Can you remove it here, and kill off any other host that has it? Mobile :)